### ISSUE TITLE
Holistically look at a11y and consider rolling in SEO 

### ISSUE THREAD

**Issue by samitron7** [GoogleForCreators/web-stories-wp#5385]
## Feature Description
a11y and seo features are found in several areas in the design panel. Our main goal should be should be to gather, organize and make surface important/critical a11y/seo, especially make it easily discoverable.

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

**Comment by dreamofabear** [comment]
Related: #5340
